generation/new_client/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:22.10
```

Comment:
Does this have all of the dependencies, including Python 3.9, etc??

Reply:
No, Python 3.9 installation and dependency download is needed.

This pull request is still my exercise to recall how to use Docker.

Reply:
Are we not dependent on a specific version of Python anymore?

Reply:
Python 3.9 installation and dependency download is needed.